Update the path to the import context directory (#3010)
neo4j/docs-operations: modules/ROOT/pages/monitoring/logging.adoc

|
|

| _dbname-yyyy-MM-dd.HH.mm.ss/import.log_ footnote:[See xref:import.adoc#import-logs[Import progress reporting] for details about the updated way of neo4j-admin import logging and reporting issues.]
| label:new[Introduced in 2025.01] Logs information about the import process when using the `neo4j-admin database import` command in a separate log file. For example, _neo4j-admin-import-2025-01-31.19.31.25.log_. +

> label:changed[Changed in 2026.03] The import log name _neo4j-admin-import-yyyy-MM-dd.HH.mm.ss.log_  is updated to _dbname-yyyy-MM-dd.HH.mm.ss/import.log_.

Author: ```suggestion
label:changed[Changed in 2026.03] The import log name _neo4j-admin-import-yyyy-MM-dd.HH.mm.ss.log_  has been changed to _dbname-yyyy-MM-dd.HH.mm.ss/import.log_, as the `neo4j-admin database import` command creates a new directory from version 2026.03.
```

> The path to it is changed from `server/logs` to `server/data/imports/`. +

Author: ```suggestion
The path to the directory is changed from `server/logs` to `server/data/imports/`. +
```

> | _neo4j-admin-import-yyyy-MM-dd.HH.mm.ss.log_ footnote:[See xref:import.adoc#import-logs[Import progress reporting] for details about the updated way of neo4j-admin import logging and reporting issues.]

Author: ```suggestion
| _dbname-yyyy-MM-dd.HH.mm.ss/import.log_ footnote:[See xref:import.adoc#import-logs[Import progress reporting] for details about the updated way of neo4j-admin import logging and reporting issues.]
```

| label:new[Introduced in 2025.01] Logs information about the import process when using the `neo4j-admin database import` command in a separate log file. For example, _neo4j-admin-import-2025-01-31.19.31.25.log_. +

> label:changed[Changed in 2026.03] The import log name and the path to it are updated to `server/data/imports/dbname-yyyy-MM-dd.HH.mm.ss/import.log` +

Author: ```suggestion
label:changed[Changed in 2026.03] The import log name _neo4j-admin-import-yyyy-MM-dd.HH.mm.ss.log_  is updated to _dbname-yyyy-MM-dd.HH.mm.ss/import.log_.
The path to it is changed from `server/logs` to `server/data/imports/`. +
```

> label:changed[Changed in 2026.03] In 2026.03, the import log name and the path to it are updated to `server/data/imports/dbname-yyyy-MM-dd.HH.mm.ss/import.log` +

Reviewer: Given that we have the label, I think we can skip "In 2026.03".

```suggestion
label:changed[Changed in 2026.03] The import log name and the path to it are updated to `server/data/imports/dbname-yyyy-MM-dd.HH.mm.ss/import.log` +
```

Reviewer: Where was the import.log before 2026.03? What was the name before? Do we have this change in the Changes in 2025-2026 series?

> label:changed[Changed in 2026.04] In 2026.04, the path is changed back to `server/logs/`.

Author: ```suggestion
label:changed[Changed in 2026.04] The path is changed back to `server/logs/`.
```

> label:changed[Changed in 2026.04] In 2026.04, the path is changed back to `server/logs/`.

Reviewer: same here
